include/mbedtls/ssl.h

```diff
@@ -787,6 +787,19 @@ typedef struct mbedtls_ssl_key_cert mbedtls_ssl_key_cert;
 typedef struct mbedtls_ssl_flight_item mbedtls_ssl_flight_item;
 #endif
 
+#if defined(MBEDTLS_SSL_PROTO_TLS1_3_EXPERIMENTAL)
+#define MBEDTLS_SSL_EARLY_DATA_NOT_SENT       0
+#define MBEDTLS_SSL_EARLY_DATA_REJECTED       1
+#define MBEDTLS_SSL_EARLY_DATA_ACCEPTED       2
+
+/**
+ * \brief get client early data state. This can only be called after handshake is completed
```

Comment:
If this is public API, it will require more extensive documentation. This can wait until the review of the rest of the PR has concluded, though.